### PR TITLE
Add and update several services

### PR DESCRIPTION
**Pull request by Gann4Life**
I have added and updated several websites to the json file, i validated it taking into account the contribution guidelines, however i haven't been able to run the project locally to make sure it actually works.

### CODE REVIEW

**Comment by tupaschoal** [comment]
>  i haven't been able to run the project locally to make sure it actually works.

That's alright, our CI will pick that up.

**Comment by Gann4Life** [comment]
Seems like i misunderstood something on the [last commit](https://github.com/jdm-contrib/jdm/pull/1158/commits/a50d5f389868c2e0b2513b6f2748ac03360546ad), any feedback would be appreciated as i'm learning how to collaborate trough github.

**Comment by Gann4Life** [comment]
[Source of the modified adfly information](https://support.adf.ly/hc/en-us/articles/207705693?input_string=account+deletion+request)

File: _data/sites.json [hunk 1]
```diff
@@ -315,12 +315,12 @@
...
-        "notes": "Contact the customer support using the contact form and request the deletion of your account.",
-        "notes_tr": "İletişim formunu kullanarak müşteri hizmetlerine başvurun ve hesabınızın silinmesini talep edin.",
-        "notes_ru": "Свяжитесь со службой поддержки, используя контактную форму, и запросите удаление вашей учетной записи.",
-        "notes_pt_br": "Entre em contato com o suporte ao cliente usando o formulário de contato e solicite a exclusão da sua conta.",
+        "url": "https://adf.ly/account/delete",
+        "difficulty": "easy",
+        "notes": "Press the button to delete your account, make sure you're logged in.",
+        "notes_tr": "Hesabınızı silmek için düğmeye basın, oturum açtığınızdan emin olun.",
+        "notes_ru": "Нажмите кнопку, чтобы удалить свою учетную запись, убедитесь, что вы вошли в систему.",
+        "notes_pt_br": "Clique no botão para excluir sua conta, verifique se você está logado.",
```
Comment (tupaschoal) [inline _data/sites.json:323]: When the notes change, we ask contributors to keep only the languages they speak, and delete the others. We don't use machine translation.

**Comment by tupaschoal** [comment]
> Seems like i misunderstood something on the last commit, any feedback would be appreciated as i'm learning how to collaborate trough github.

All seems well now, so I assume you figured it out (?)

File: _data/sites.json (hunk 1 above)
Comment (Gann4Life) [reply _data/sites.json:323]: Understood, thanks.
It's better to remove the languages i don't speak then?
Said in other way, only keep the english notes and remove the rest.

Comment (tupaschoal) [reply _data/sites.json:323]: Yes, please!

**Review by tupaschoal — APPROVED** [review]

**Comment by tupaschoal** [comment]
Thanks for your contribution!

**Pull request merged**